src/django/requirements.txt

```diff
@@ -1,3 +1,5 @@
+Django==1.10.3
```

Comment:
In the same vein, might want to note with a comment which are application specific requirements and which are artifiacts of having to duplicate the parent container's requirements.txt file.